addon-test-support/helpers.js

```diff
@@ -111,6 +114,7 @@ function buildKeyboardEvent(type, options = {}) {
 }
 
 export function click(selector, options = {}) {
+  selector = `${rootElement} ${selector}`;
```

Comment:
We should generally prefer to avoid mutating an argument name like this (it makes debugging / stepping in chrome difficult).  I'd suggest something like this instead:

```js
export function click(_selector, options = {}) {
  let selector = `${rootElement} ${_selector}`;
}
```